Resources/config/services.yml

```diff
@@ -1,10 +1,19 @@
+parameters:
+    grossum_menu_tree_depth: 32
+
 services:
     grossum_menu.menu.entity.manager:
-        class: %grossum_menu.menu.manager.class%
+        class: %grossum_menu.menu.entity.manager.class%
```

Comment:
I think, we should rename it as `grossum_menu.entity_manager.base_menu.class` or something like this